### PR TITLE
Move locale bn-BD(Bengali Bangladesh) to bn(Bengali)

### LINKED ISSUE #8336: Move locale bn-BD to bn
We are going to merge the two bn-* locales into one to fully utilize the community resources. The frontend has only bn-BD. We want to sync up with the rest of the localized products/projects. The plan is

Dev:
- AMO Frontend creates a bn folder at repo
- Move bn-BD repo content to bn repo and confirm it's all there
- Remove bn-BD repo

Pontoon (l10n):
- Pontoon dev migrates the data to bn (this includes contributors histories)
- L10n PM confirms the update and closes this issue.

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #8336

### CODE REVIEW

**Comment by championshuttler** [comment]
Before: 

image


After:

image

File: src/core/languages.js [hunk 1]
```diff
@@ -50,15 +50,6 @@ export const unfilteredLanguages = {
...
   },
-  'bn-BD': {
-    English: 'Bengali (Bangladesh)',
-    native:
-      '\u09ac\u09be\u0982\u09b2\u09be (\u09ac\u09be\u0982\u09b2\u09be\u09a6\u09c7\u09b6)',
-  },
-  'bn-IN': {
-    English: 'Bengali (India)',
-    native: '\u09ac\u09be\u0982\u09b2\u09be (\u09ad\u09be\u09b0\u09a4)',
-  },
```
Comment (willdurand) [inline src/core/languages.js]: I don't think this should be removed.

Comment (championshuttler) [reply src/core/languages.js]: @willdurand I can delete it but i guess they are not gonna use in future then why we need to add them here?

Comment (willdurand) [reply src/core/languages.js]: I have no strong opinion about it but I'd say we might still need that in the future.

Comment (championshuttler) [reply src/core/languages.js]: Ok updating no worries :)

Comment (championshuttler) [reply src/core/languages.js]: Hmm 👀 , updating the patch :)

**Pull request merged**